docs/hugo/content/contributing/create-a-new-release.md

```diff
@@ -22,6 +22,23 @@
    ```
 9. Make sure they deploy successfully - check in the portal as well.
 
+# Creating and testing Helm chart for new release
+1. Create a new branch from `<NEW_RELEASE_TAG>` HEAD
+2. Generate helm manifest for new release: `task controller:gen-helm-manifest`
+3. Check the version in `/v2/charts/azure-service-operator/Chart.yaml` if matches with the latest release tag.
+4. Install helm chart:
+```
+helm install --set azureSubscriptionID=$AZURE_SUBSCRIPTION_ID \
+--set azureTenantID=$AZURE_TENANT_ID \
+--set azureClientSecret=$AZURE_CLIENT_SECRET \
+--set azureClientID=$AZURE_CLIENT_ID \
+asov2 -n azureserviceoperator-system --create-namespace ./v2/charts/azure-service-operator/.
+```
+5. Wait for the chart installation.
+6. Ensure that the chart is installed successfully.
```

Comment:
minor: What's the way to do this? Presumably something like step 8 in the instructions above? (Actually install resource and make sure it works?)

Reply:
Done!